css-four-column-layout/styles.css

```diff
@@ -0,0 +1,79 @@
+body {
+  background-color: rgb(247 248 250);
+}
+
+.column-full {
+  width: 1280px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  font-family: Barlow, Arial, sans-serif;
+  text-align: center;
+  font-size: 20px;
+}
+
+.container {
+  font-family: Barlow, Arial, sans-serif;
+  width: 1280px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.four-column-row {
+  display: flex;
+  width: 1080px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.column-one {
+  width: 25%;
+}
+
+.column-two {
+  width: 25%;
+}
+
+.column-three {
+  width: 25%;
+}
+
+.column-four {
+  width: 25%;
+}
```

Comment:
since each of these classes is doing the same thing, I would just create one `column-fourth` class and use that on each of the elements to reduce code redundancy